mod_reforged/hooks/config/strings.nut

```diff
@@ -2242,12 +2242,15 @@ foreach (vanillaDesc in vanillaDescriptions)
 		}]
  	}),
 	RF_WhirlingDeath = ::UPD.getDescription({
- 		Fluff = "Use the arc of your flailhead to target weak spots in your opponents' armor!",
+ 		Fluff = "Create a whirlwind of death with the spinning head of your flail!",
  		Requirement = "Flail",
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"An additional random " + ::MSU.Text.colorGreen("0-25%") + " of damage ignores armor."
+				"Every attack with flails increases your current [Reach|Concept.Reach] by " + ::MSU.Text.colorGreen("+1") + " until your next [turn|Concept.Turn].",
+				"With two-handed flails, when attacking a target who has lower current [Reach|Concept.Reach] than you, gain " + ::MSU.Text.colorGreen("+10%") + " chance to hit the head per difference in current [Reach|Concept.Reach] between you and the target.",
+				"During your [turn|Concept.Reach], when attacking with one-handed flails, perform a free extra attack with " + ::MSU.Text.colorRed("50%") + " less damage against a different adjacent enemy who has lower current [Reach|Concept.Reach] than you.",
```

Comment:
`[turn|Concept.Reach]` - Reach is the wrong concept for turn

Reply:
Lol, nice catch!